katello/katello/restore.rb

```diff
@@ -19,6 +19,7 @@ def initialize(foreman_proxy_content, program, accepted_scenarios=nil)
       @program = program
 
       @confirmed = false
+      @split_pulp_tar_script = '/usr/share/katello/katello-backup-rotate-tar.sh'
```

Comment:
since this is used in both scripts, it seems like a good candidate for helper.rb. (but also keeping in mind the comment about `/usr/share/katello` directory)

Reply:
I'll take a stab at this